ui/src/common/__tests__/utils.test.js

```diff
@@ -327,14 +327,21 @@ describe('utils', () => {
 
     it('convert http network error', () => {
       const error = {
-        response: null,
+        message: 'Network Error',
       };
       const result = httpErrorToActionPayload(error);
       const expected = {
         status: 'network',
       };
       expect(result).toEqual(expected);
     });
+
+    it('throw other errors', () => {
+      const error = 'whatever';
+      const result = httpErrorToActionPayload;
```

Comment:
we don't need this line, you can directly call `httpErrorToActionPayload` on the next line.